### ISSUE TITLE
[DOC]Readme中有些问题

### ISSUE THREAD

**Issue by akihoni** [milvus-io/milvus#100]
1. release notes 链接无效。

2. Step 1 Install dependencies

   ```
   $ cd [Milvus sourcecode path]/core
   ./ubuntu_build_deps.sh
   ```

   第二行是不是缺了一个`$`？

**Comment by JinHai-CN** [comment]
Yes, we are correcting the issue.

**Comment by jielinxu** [comment]
@akihoni Thank you for your observation. The issue will be fixed in #102 .

**Comment by jielinxu** [comment]
Done.
